packages/svelte/src/runtime/store/index.js

```diff
@@ -26,7 +26,7 @@ export function readable(value, start) {
  * Create a `Writable` store that allows both updating and reading by subscription.
  * @template T
  * @param {T} value initial value
- * @param {import('./public.js').StartStopNotifier<T>} start
+ * @param {import('./public.js').StartStopNotifier<T>} [start]
```

Comment:
this is not optional in `master`. was v3 wrong?

https://github.com/sveltejs/svelte/blob/3bc791bcba97f0810165c7a2e215563993a0989b/src/runtime/store/index.ts#L69